eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/props/AppProperties.java

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Please remove these default headers and let maven replace them with the auto generated headers (license-maven-plugin).